### ISSUE TITLE
Server logger (format, output, etc) does not work

### ISSUE THREAD

**Issue by ongietorri** [bokeh/bokeh#6699]
hello

Anyone had luck redirecting the server logs to a file?

Using Bokeh 0.12.6 (latest), I can't seem to be able to use the bokeh logger. 

I tried using the [basicConfig ](http://bokeh.pydata.org/en/latest/docs/reference/util.html#bokeh-util-logconfig)wrapper but id does not seem to work.

```
from bokeh.util import logconfig
logconfig.basicConfig(filename="output.log")
```

I also tried creating another standard logger without using Bokeh (e.g. with logging module), but it does not work either.

Here's a minimal example where I try do directly use the `bokeh_logger` :

```
from bokeh.io import curdoc
from bokeh.util.logconfig import basicConfig, bokeh_logger as bl
import logging

basicConfig(level=logging.DEBUG, filename='output.log')
bl.info('info......')
bl.error('error.......')
bl.warning('Warning.......')

doc = curdoc()
```

Am I missing something? Maybe it's a bug.

Thanks

**Comment by ongietorri** [comment]
Seems to be fixed in 0.12.13

**Comment by chesucr** [comment]
Hello! Are your examples working then? They are not working for me for the 0.12.13 version

There are two options when the bokeh server is run from terminal:
```
  --log-level LOG-LEVEL
                        One of: trace, debug, info, warning, error or critical
  --log-format LOG-FORMAT
                        A standard Python logging format string (default:
                        '%(asctime)s %(message)s')
```
I suggest adding a third option to set where to store or send the output:

* `stdout`. Should this be the default value?
* A file path where to store the log file

The basicConfig is [defined here](https://github.com/bokeh/bokeh/blob/master/bokeh/util/logconfig.py). The `stdout` stream could be set like this: 
```
import sys
logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
```
